include/fmt/format.h

```diff
@@ -901,14 +901,14 @@ struct FMT_EXTERN_TEMPLATE_API uint128_wrapper {
 #if FMT_USE_INT128
   uint128_t internal_;
 
-  uint128_wrapper(uint64_t high, uint64_t low) FMT_NOEXCEPT
+  FMT_CONSTEXPR uint128_wrapper(uint64_t high, uint64_t low) FMT_NOEXCEPT
```

Comment:
Please use `constexpr` instead of `FMT_CONSTEXPR` here and below since this looks compatible with C++11 constexpr.

Reply:
Ahh thanks yup it is compatible with C++11 constexpr. Done.